frontend/src/component/common/DialogFormTemplate/ConfigButtons/ToggleConfigButton.tsx

```diff
@@ -0,0 +1,52 @@
+import { ButtonLabel, StyledTooltipContent } from './ConfigButton.styles';
+import { TooltipResolver } from 'component/common/TooltipResolver/TooltipResolver';
+import { Button } from '@mui/material';
+import type { ReactNode } from 'react';
+
+type ToggleConfigButtonProps = {
+    onClick: () => void;
+    currentValue: boolean;
+    label: string;
+    icon: ReactNode;
+    labelWidth?: string;
+    additionalTooltipContent?: ReactNode;
```

Comment:
Is this needed?

Reply:
No, you're right. Just forgot to delete it. Good catch